### PR TITLE
one-team, shadow, 3 column template issues

### PR DESCRIPTION
**Pull request by tal-athena**
one-team component added.
multi grey layers are used for one-team component shadow
3 column template is done.

### CODE REVIEW

**Comment by paulchrisluke** [comment]
there are some issues with outlook here. Provided email on acid credentials to @tal-athena please continue, looking good!

File: src/partials/article-bold.html [hunk 1]
```diff
@@ -31,7 +31,7 @@
         <!--[if !mso]>-->
         <th
           height="26"
-          style="height:26px; background:url(https://res.cloudinary.com/whynotearth/image/upload/e_colorize,co_rgb:{{boxColorHex}}/Volkswagen/newsletter1/line-160px-2px-96dpi_rdzowd.png) repeat-x;"
+          style="height:26px; background-image:url(https://res.cloudinary.com/whynotearth/image/upload/e_colorize,co_rgb:{{boxColorHex}}/Volkswagen/newsletter1/line-160px-2px-96dpi_rdzowd.png)"
```
Comment (mort3za) [inline src/partials/article-bold.html:34]: Is this change related to the issue?

File: src/assets/scss/template/_newsletter-1.scss [hunk 2]
```diff
@@ -60,6 +60,8 @@
 
 // article content
 .article-content {
+
+  box-shadow: 0 3px 4px -2px rgba(0, 0, 0, 0.25);
```
Comment (mort3za) [inline src/assets/scss/template/_newsletter-1.scss:64]: This won't work in Outlook

Comment (mort3za) [reply src/assets/scss/template/_newsletter-1.scss:64]: Anyway, don't put time on shadows, I'm working on shadows on a separate branch.
